src/features/percentileCalculator/components/PercentileResults.tsx

```diff
@@ -45,12 +45,12 @@ export const PercentileResults = () => {
   return (
     <div data-testid="percentile-result-tables" className={classes.root}>
       <GridContainer className={classes.gridContainer}>
-        {stationResults}
         {isMoreThanOneResult && (
-          <GridItem lg={6}>
+          <GridItem lg={12}>
```

Comment:
`md` prop needs to be set to 12 as well. If you look at `Grid.tsx`, you can see the component's implementation.
![image](https://user-images.githubusercontent.com/36203857/80743864-96716300-8ad2-11ea-85a5-b5c4fece7e7a.png)